compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/ConstantValueResolver.java

```diff
@@ -215,6 +222,25 @@ private BLangConstantValue calculateConstValue(BLangConstantValue lhs, BLangCons
         return new BLangConstantValue(null, this.currentConstSymbol.type);
     }
 
+    private BLangConstantValue calculateConstValue(BLangConstantValue value, OperatorKind kind) {
+
```

Comment:
Let's remove the unwanted newline in other places as well

Reply:
will reformat all